.travis.yml

```diff
@@ -1,96 +1,60 @@
 language: cpp
+
+dist: trusty
```

Comment:
Travis supports two distros, trusty is the more recent / up-to-date. There's no updated needed, as travis ships the updates.